docs/source/contributing/testing-vitest.md

```diff
@@ -0,0 +1,96 @@
+---
+myst:
+  html_meta:
+    "description": "We use Vitest for unit testing in Volto. The popular @testing-library/react is also available for writing your tests. For every feature or component, a unit test is mandatory in Volto core."
+    "property=og:description": "We use Vitest for unit testing in Volto. The popular @testing-library/react is also available for writing your tests. For every feature or component, a unit test is mandatory in Volto core."
+    "property=og:title": "Testing Volto with Vitest"
+    "keywords": "Volto, Plone, frontend, React, testing, Vitest"
+---
+
+# Testing with Vitest
+
+We use `Vitest` for unit testing in Volto.
+The popular `@testing-library/react` is also available for writing your tests.
+For every feature or component, a unit test is mandatory in Volto core.
+
+## Vitest configuration
+
+Vitest is configured in `package.json` under the `vitest` key. 
```

Comment:
All files should have the `{file}` directive. It will render appropriate semantic HTML. I'll do just this one, but please do all other files.

```suggestion
Vitest is configured in {file}`package.json` under the `vitest` key. 
```